addons/exact-count/user.js

```diff
@@ -3,47 +3,68 @@ export default async function ({ addon, global, console }) {
   let user1 = url.substring(30, 100);
   let username = user1.substring(0, user1.indexOf("/"));
   let dalength = document.getElementsByClassName("box-head").length;
+  var boxNumber = 1;
+  getUserData(
+    {
+      boxheadName: "Shared Projects",
+      url: "projects",
+    },
+    function () {
```

Comment:
Why callbacks?? Just make it a function below the other, please

Reply:
I tried, but the API gives the data at random times and it messed up everything so I was trying to give it an order... is there a different solution?

Reply:
You should pass `boxNumber` to each function then

Reply:
But what if that person has not joined any studio or followed any? Then those boxes would disappear and the count would be wrong.

Reply:
You should first check what rows exist - then, request the counts.